internal/ingress/annotations/annotations.go

```diff
@@ -86,7 +86,7 @@ type Ingress struct {
 	CorsConfig           cors.Config
 	CustomHTTPErrors     []int
 	DefaultBackend       *apiv1.Service
-	//TODO: Change this back into an error when https://github.com/imdario/mergo/issues/100 is resolved
+	// TODO: Change this back into an error when https://github.com/imdario/mergo/issues/100 is resolved
```

Comment:
I think this comment is leading, I have no idea what this means. Go ahead and remove it :)